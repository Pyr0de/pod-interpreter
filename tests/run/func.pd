func a() {
	print "hello";
}

a();

func b(m,n) {
	print m;
	print n;
}

b(10 + 1, true);

func rec(p) {
	if p > 3 {
		print p;
	}else {
		rec(p+1);
	}
}
rec(0);


{
	let k = 1;

	func x() {
		print k;
	}

	{
		func y() {
			x();
		}
		y();
	}
	y();
}
