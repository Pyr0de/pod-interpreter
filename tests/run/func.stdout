ExitCode: 0
hello
11
true
4
